pokedex/db/load.py

```diff
@@ -193,7 +193,35 @@ def load(session, tables=[], directory=None, drop_tables=False, verbose=False, s
 
     print_start('Creating tables')
     for n, table in enumerate(table_objs):
-        table.create(bind=engine)
+        try:
+            table.create(bind=engine)
+        
+        # Exceptions for handling the error thrown when trying to load
+        # the database with a table that already exists.
+        except (
+            sqlalchemy.exc.OperationalError,  # Exception used for SQLite
+            sqlalchemy.exc.ProgrammingError,  # Exception used for PostgreSQL
+            sqlalchemy.exc.InternalError      # Exception used for MySQL
+            ) as error:
+
+            if "already exists" in str(error.orig):
+                print("\n\nERROR:  The table '{}' already exists in the database. "
+                    "Did you mean to use 'pokedex load -D'".format(table))
+                sys.exit(1)
+
+            # If it happens to be some other error but raised by the same 
+            # exception, then an unexpected error message is sent with 
+            # the error included
+            else:
+                print("\n\n UNEXPECTED ERROR: ", error)
```

Comment:
i wonder if this should include some context about what we were doing that caused the error. e.g.,

    ERROR: creating table "pokemon_species_names": sqlalchemy.exc.WhateverError: invalid syntax or whatever

Or do the exceptions that sqlalchemy gives us already have enough context?

Reply:
SQLAlchemy already shows a lot of information. Here is an example of the error message (this is the message if the exception in line 202 was removed)
```
UNEXPECTED Error:   (sqlite3.OperationalError) table languages already exists [SQL: '\nCREATE TABLE languages
(\n\tid INTEGER NOT NULL, \n\tiso639 VARCHAR(79) NOT NULL, \n\tiso3166 VARCHAR(79) NOT NULL, \n\tidentifier VARCHAR(79) NOT NULL, \n\tofficial BOOLEAN NOT NULL, \n\t"order" INTEGER, \n\tPRIMARY KEY (id), \n\tCHECK (official IN (0, 1))\n)\n\n'] (Background on this error at: http://sqlalche.me/e/e3q8)
```
The UNEXPECTED ERROR else statement needs to be there since some other exception (not a table already exists error) could happen and the program would continue to execute and give a `UNIQUE constraint failed: languages.id` error. That error makes it a lot harder to debug.